diff --git a/node_modules/expo/src/winter/url.ts b/node_modules/expo/src/winter/url.ts
index e8a66c0..c6ba502 100644
--- a/node_modules/expo/src/winter/url.ts
+++ b/node_modules/expo/src/winter/url.ts
@@ -106,7 +106,7 @@ URL.revokeObjectURL = function revokeObjectURL(_url) {
 
 URL.canParse = function canParse(url: string, base?: string): boolean {
   try {
-    URL(url, base);
+    new URL(url, base);
     return true;
   } catch {
     return false;
